### ISSUE TITLE
Smarthome support

### ISSUE THREAD

**Issue by gromdroid** [tartanguru/alexa-assistant#53]
Hi,

I don't know if this is the right place, but is it possible to ask google assistant to play a video on my chromecast through this skill? Or is this only for search queries?

**Comment by tartanguru** [comment]
I don't believe this is supported by the Google API. I don't own a Chromecast so can't confirm

**Comment by dmacx** [comment]
First off, many many thanks to @tartanguru for all his work on the google skill - I finally got round to enabling this and am well impressed!
@gromdroid -
There is another github with a skill that allows Alexa to send commands to chromecasts:
[https://github.com/Pizzaface/Alexa-Chromecast-Skill-2.0](https://github.com/Pizzaface/Alexa-Chromecast-Skill-2.0).
I have got it working, requires a Raspberry Pi or similar to function.
